.github/workflows/mds-spec-change-issue.yml

```
@@ -0,0 +1,109 @@
+
+          {
+            echo "files<<EOF"
+            echo "$CHANGED"
+            echo "EOF"
+          } >> "$GITHUB_OUTPUT"
+          echo "skip=false" >> "$GITHUB_OUTPUT"
+
+      - name: Build issue body
+        if: steps.collect.outputs.skip != 'true'
+        id: body
+        run: |
```

actionlint (1.7.12)

[error] "github.event.head_commit.author.name" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)